Gemfile

```diff
@@ -8,7 +8,4 @@ gem "rake"
 gem "rake-compiler"
 gem "test-unit"
 gem "ruby_memcheck", platform: %i[mri truffleruby mswin mingw x64_mingw]
-
-if RUBY_ENGINE == 'ruby' and ENV["YARP_FFI_BACKEND"]
-  gem "ffi"
-end
+gem "ffi"
```

Comment:
One side effect of this is it also installs the latest version of the ffi gem on JRuby and TruffleRuby.
That's not really representative for the `gem install` test, because ffi is not a dependency of the yarp gem, because JRuby/TruffleRuby ship with ffi anyway.
And it seems more valuable to test with the version they ship than latest, as e.g. latest might have more features but we shouldn't rely on them.
So we should add `platform: :mri` here, to test like a end user would (they wouldn't install ffi as a dependency of yarp).

Reply:
https://github.com/ruby/yarp/pull/1254